test/hello-world_test.coffee

```diff
@@ -1,21 +1,20 @@
 Helper = require('../src/index')
-helper = new Helper('./scripts')
+helper = new Helper('./scripts/hello-world.coffee')
```

Comment:
Should we load only script related to the test?
Nobody wants to load all scripts automatically?

I accepted PR to load individual script (#1),
but I prefer load all scripts because Hubot load all scripts and may cause regexp conflict.

What do you think?